server/db/build-rhythms.sql

```diff
@@ -32,13 +32,20 @@ CREATE TABLE rhythms (
-INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Graj', 'French', 1700, 2020, 'https://drive.google.com/file/d/1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm/view','The graj accompanies the agricultural work');
- 
\ No newline at end of file
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Graj', 'French', 1700, 2020, 'https://drive.google.com/file/d/1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm','The graj accompanies the agricultural work');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Mende', 'French', 1850, 2020, 'https://drive.google.com/uc?id=1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm','The mendé would have been the last rhythm to arrive in the islands, with the "Congos" under contract after the abolition, and symbolizes the collective celebration of carnival');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Pajabel', 'French', 1700, 2020, 'https://drive.google.com/uc?id=1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm','The padjabèl is the cane cutting dance');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Lewoz', 'French', 1700, 2020, 'https://drive.google.com/uc?id=1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm','The léwôz is the war rhythm, used to give rhythm the attacks against the plantations, but was also an incantation dance');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, audio, description) VALUES(2, 'Tumbla', 'French', 1700, 2020, 'https://drive.google.com/uc?id=1MoZXjAO2cqPU3IssoP40PYrAsshT4PVm','The toumblak, like the kaladja, deals with the love theme, belly dance, fertility dance');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, video, description) VALUES(3, 'Second Line', 'English', NULL, NULL, 'https://www.youtube.com/watch?v=QpKiuVVXXeA','Description needed');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, video, description) VALUES(4, 'One Step', 'English', 1870, 2020, 'https://www.youtube.com/watch?v=rF9y2qIbUqw','One step has a single walking rhythm');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, video, description) VALUES(4, 'Two Step', 'English', 1900, 2020, NULL,'Two step combines two rhythms, typically quicks and slows');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, video, description) VALUES(4, 'Tresillo', 'English', 1920, 1960, NULL,'Tresillo inherits from the Habanera rhythm and also has later links to the bo-diddly');
+INSERT INTO rhythms (rhythm_code_id, rhythm, language, year_start, year_end, video, description) VALUES(5, 'DC Hand Dancing', 'English', NULL, NULL, NULL,'Description needed');
```

Comment:
Better to just show `NULL` than `Description needed`

Reply:
Done, thank you.